doc/compiler/jitserver/Usage.md

```diff
@@ -79,14 +84,14 @@ $ java -XX:+UseJITServer:server=example.com
 #### Port
 By default, communication occurs on port 38400. You can change this by specifying the `port` suboption as follows:
 ```
-$ java -XX:StartAsJITServer -XX:JITServerPort=1234
+$ jitserver -XX:JITServerPort=1234
 $ java -XX:+UseJITServer -XX:JITServerPort=1234 MyApplication
 ```
 
 #### Timeout
 If your network connection is flaky, you may want to specify a timeout. By default there is no timeout. Timeout is given in milliseconds. Client and server timeouts do not need to match, but there's usually no reason for them to differ.
```

Comment:
By default there is a 30000 ms timeout at the server and 2000 ms timeout at the client

Reply:
"there's usually no reason for them to differ." is incorrect. Typically the timeout at the server can be larger; it can afford to wait because there is nothing else to do anyway. Waiting too much at the client can be detrimental because the client has the option of compiling locally and make progress.